core/src/main/scala/cats/replicateH/replicateH.scala

```diff
@@ -1,24 +1,26 @@
 package cats.replicateH
 
+import cats.Applicative
```

Comment:
nit: is this import used?

Reply:
Ah good catch. I was going to add the support here instead of within `Sequencer` at first. I'll fix this.